src/helpers/Initialization.tsx

```diff
@@ -0,0 +1,16 @@
+﻿import MyDataHelps from '@careevolution/mydatahelps-js';
+import {useEffect} from 'react';
+
+const debounce = require('lodash.debounce');
+
+export function useInitializeView(initialize: () => void): void {
```

Comment:
This is a very handy shortcut; I've copy/pasted this in a ton of places in another application without thinking to make it a custom hook.  Maybe two small suggestions:
1) I didn't instinctively know what the hook was doing where you used it in the AppDownload widget until I came down here.  I know it's a long name, but perhaps `useApplicationDidBecomeVisibleInitialization`?
2) It might be helpful to have a second parameter to the hook which is the array of dependencies that passed to useEffect?

Reply:
I'm not opposed to renaming it, but I'm hesitant to rename it something to do with a specific event.  Similar to your suggestion about the array of dependencies, I had considered allowing folks to pass in a collection of events too.  I have a version of this in a couple projects, and one of them registers two different events.

I'll see if I can update it to support both of those customization scenarios.

Reply:
I've added the ability to pass in additional events as well as dependencies.

Let me know if you have any other naming suggestions given the latest functionality.

I went with `useInitializeView` because I wanted it to be `initializeView`, but being a hook, it needed the `use` prefix.